src/components/KSelect/KSelect.vue

```diff
@@ -466,7 +466,7 @@ const handleAddItem = (): void => {
   const pos = (selectItems.value?.length || 0) + 1
   const item: SelectItem = {
     label: sanitizeInput(filterQuery.value),
-    value: uuidv4(),
+    value: useUniqueId(),
```

Comment:
technically this composable should only be called at the root; should you set this value outside?

Reply:
Why is this composable should only be called at the root?

Reply:
Composables, by design, should only be called from the root of the setup function

Reply:
Huh, I didn't know that.. we can't move this one to the root because it generates random id for each added item (it's inside `handleAddItem` function). Should I just use `nanoid` straight up instead?

Reply:
it's probably fine here considering what this one is doing